### PR TITLE
Update how-to documentation

### PR DESCRIPTION
**Pull request by erinecon**
### Overview

Edits to four of the how-to guides

### Rationale

Improve the formatting, remove passive voice, other general edits

### Juju Events Changes

None

### Module Changes

None

### Library Changes

None

### Checklist

- [X] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [X] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [X] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [X] The documentation is generated using `src-docs`
- [X] The documentation for charmhub is updated
- [X] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: docs/how-to/configure-a-proxy.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # How to configure a proxy
 
-This charm supports several operations that require access to external resources, such as customization or plugins installation, so your environment may not be able to reach those URLs unless through a proxy. The proxy can be configured via the [juju model configuration](https://juju.is/docs/juju/manage-models#heading--configure-a-model) setting the [HTTP proxy keys](https://juju.is/docs/juju/list-of-model-configuration-keys).
\ No newline at end of file
+This charm supports several operations that require access to external resources, such as customization or plugins installation, so your environment will only be able to reach those URLs through a proxy. The proxy can be configured via the [juju model configuration](https://juju.is/docs/juju/manage-models#heading--configure-a-model) by setting the [HTTP proxy keys](https://juju.is/docs/juju/list-of-model-configuration-keys).
```
Comment (varshigupta12) [inline docs/how-to/configure-a-proxy.md:3]: Nit: Let's keep "may only" instead of "will only"

Comment (erinecon) [reply docs/how-to/configure-a-proxy.md]: Sounds good, thanks for catching!

**Review by arturo-seijas — APPROVED** [review]

**Review by amandahla — APPROVED** [review]
LGTM

**Comment by erinecon** [comment]
Closing the PR because I did not sign the original commit; will reopen today

**Pull request closed without merging**